model/github_hook.go

```diff
@@ -144,17 +144,17 @@ func GetExistingGithubHook(ctx context.Context, settings evergreen.Settings, own
 	}
 	defer util.PutHTTPClient(httpClient)
 	client := github.NewClient(httpClient)
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	newCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
I didn't change this because of the bug, rather to match the convention of createHook.